.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,435 @@
+          mkdir -p binaryData/Organization_X
+          unzip test/dataset/test-dataset.zip -d binaryData/Organization_X/
+          sudo chmod -R "0777" binaryData/Organization_X
+
+          mkdir -p fossildb/data && sudo chmod -R "0777" fossildb/data
+          mkdir -p fossildb/backup && sudo chmod -R "0777" fossildb/backup
+      - name: Start Docker Instance
+        if: ${{ github.ref_name != 'master' }}
+        run: |
+          docker compose -f .github/docker-compose.yml -p "webknossos" up -d
+        env:
+          DOCKER_TAG: ${{ steps.configure.outputs.docker_tag }}
```

actionlint (1.7.4)

property "configure" is not defined in object type {}

(expression)